torch_geometric/transforms/feature_prop.py

```diff
@@ -0,0 +1,83 @@
+import torch
+from torch_sparse import SparseTensor, matmul
+
+from torch_geometric.data import Data
+from torch_geometric.data.datapipes import functional_transform
+from torch_geometric.nn.conv.gcn_conv import gcn_norm
+from torch_geometric.transforms import BaseTransform
+from torch_geometric.typing import OptTensor
+
+
+@functional_transform('feature_propagation')
+class FeaturePropagation(BaseTransform):
+    r"""The feature propagation operator from the
+    `"On the Unreasonable Effectiveness of Feature
+    propagation in Learning on Graphs with Missing Node Features"
+    <https://arxiv.org/abs/2111.12128>`_ paper
+
+    .. math::
+        \mathbf{X}^{\prime} = \mathbf{D}^{-1/2} \mathbf{A}
+        \mathbf{D}^{-1/2} \mathbf{X},
+
+        \mathbf{X}_{k} = \mathbf{X}^{\prime}_{k}
+
+
+    where missing node features are inferred by known features via propagation.
+
+    .. note::
+
+        For an example of using :class:`~torch_geometric.transforms.
+        FeaturePropagation`, see `examples/feature_propagation.py
+        <https://github.com/pyg-team/pytorch_geometric/blob/master/examples/
+        feature_propagation.py>`_.
+
+    Args:
+        num_iterations (int): The number of propagations. (default: :obj:`40`)
+        missing_mask (Tensor, optional): The mask indicating where
+            node features are missing. (default: :obj:`None`)
```

Comment:
```suggestion
        missing_mask (Tensor, optional): The mask indicating where
            node features are missing, of shape `[num_nodes, channels]`. (default: :obj:`None`)
```
Add shape information.